pkg/plugins/handlers.go

```diff
@@ -167,6 +167,9 @@ func (p *PluginsHandler) proxyPluginRequest(requestURL *url.URL, pluginName stri
 	}
 	defer resp.Body.Close()
 
+	// Make sure to copy status code from the plugin service response
+	w.WriteHeader(resp.StatusCode)
```

Comment:
I think this needs to be moved to after line 180, so that the headers can be copied to the response first.

Reply:
@KevinFCormier is right. This needs to be moved before `io.Copy`